File: JavaScript_Courses/JavaScript_Basics/Preparation/03_Conditional_Statements_Advanced_Exercise/01_Cinema.js
function cinema(input) {
    let premierePrice = 12.00;
    let normalPrice = 7.50;
    let discountPrice = 5.00;

    let ticketType = input[0];
    let rows = Number(input[1]);
    let cols = Number(input[2]);

    let income = 0;

    if (ticketType == "Premiere") {
        income = rows * cols * premierePrice;
    } else if (ticketType == "Normal") {
        income = rows * cols * normalPrice;
    } else if (ticketType == "Discount") {
        income = rows * cols * discountPrice;
    }

    console.log(income.toFixed(2) + " leva");
}

// cinema(["Premiere", "10", "12"]);

File: Python_Courses/Python_Basics/Python_PB_2023_04/03_Conditional_Statements_Advanced_Exercise/01_Cinema.py
# # Solution 1
# PREMIERE_PRICE = 12.00
# NORMAL_PRICE = 7.50
# DISCOUNT_PRICE = 5.00
#
# screening_type = input()
# rows = int(input())
# cols = int(input())
#
# screening_capacity = rows * cols
#
# price = 0
# if screening_type == "Premiere":
#     price = PREMIERE_PRICE
# elif screening_type == "Normal":
#     price = NORMAL_PRICE
# elif screening_type == "Discount":
#     price = DISCOUNT_PRICE
#
# total_income = screening_capacity * price
# print(f"{total_income :.2f} leva")

# Solution 2
prices = {
    "Premiere": 12.00,
    "Normal": 7.50,
    "Discount": 5.00,
}

screening_type = input()
rows = int(input())
cols = int(input())

screening_capacity = rows * cols
price = prices[screening_type]

total_income = screening_capacity * price
print(f"{total_income :.2f} leva")
